src/dodal/devices/apple2_undulator.py

```diff
@@ -186,25 +218,14 @@ def __init__(self, prefix: str, infix: str, name: str = ""):
         self.user_setpoint = epics_signal_w(str, fullPV + "SET")
         self.user_setpoint_demand_readback = epics_signal_r(float, fullPV + "DMD")
 
-        fullPV = fullPV + "MTR"
+        motor_PV = fullPV + "MTR"
         with self.add_children_as_readables(StandardReadableFormat.HINTED_SIGNAL):
-            self.user_setpoint_readback = epics_signal_r(float, fullPV + ".RBV")
+            self.user_setpoint_readback = epics_signal_r(float, motor_PV + ".RBV")
```

Comment:
@Relm-Arrowny I could instead make everything use `user_readback` but this changes the name of the signal when it is `read()` I wasn't sure if the name was important so left it.

Reply:
Ok, if the motor bit is the controversial one I'll revert that and make a follow up issue